resources/views/site/blocks/feature_block.blade.php

```diff
@@ -112,12 +110,20 @@
                                             'xlarge' => 38,
                                         ))))
                                 @endcomponent
+                                @if ($item->type === 'exhibition' && $item->is_now_open || $item->is_closing_soon || $item->is_ongoing)
```

Comment:
The way this will be processed is "if ($item->type === 'exhibition' && $item->is_now_open) or ($item->is_closing_soon) or ($item->is_ongoing)." So those last two will be evaluated independently of the item type check. (@zachgarwood check me if I'm misunderstanding here).

To solve this, and to simply be more clear in your coding intentions, wrap the three `is_*` checks in parentheses since you need one of those three things to be true in combination with the item type being `exhibition`:

```
@if ($item->type === 'exhibition' && ($item->is_now_open || $item->is_closing_soon || $item->is_ongoing))
```